simulation/agents.py

```diff
@@ -118,6 +120,19 @@ def wealth_percentile(self) -> float:
             self.property_value,
         )
 
+    @property
+    def discount_rate(self) -> float:
+
+        return max(
+            1
+            - self.get_weibull_value_from_percentile(
```

Comment:
Why do we have `1 -` here and not just the distribution of discount rates as a weibull?

Reply:
E.g. the 90th wealth percentile corresponds to the 10th percentile of discount rates, because of the studied effect of discount rates being inversely related to income or wealth. The theory is that lower income households do not/cannot value future cashflows as highly as near-term ones, due to more pressing immediate financial needs -- therefore apply higher 'discount' rates to these cashflows.